index.html

```diff
@@ -0,0 +1,91 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8" />
+    <meta http-equiv="X-UA-Compatible" content="IE=edge" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <title>Wuyep Abdul</title>
+    <link rel="stylesheet" href="./style.css" />
+  </head>
+  <body>
+    <header class="nav-bar align-center d-flex space-between">
+      <a class="padding-round">John Doe</a>
+      <nav class="bars"><img src="./images/bars.png" alt="image" /></nav>
+    </header>
+
+    <main>
+      <section class="headline d-flex align-center">
+        <div class="headline-ellipse">
+          <header>
+            <h1>Laura Jong</h1>
+            <h2>Software Developer</h2>
+          </header>
+        </div>
+
+        <div class="intro align-center">
+          <p>
+            I can help you build a product , feature or website Look through
+            some of my work and experience! If you like what you see and have a
+            project you need coded, don’t hestiate to contact me.
+          </p>
+          <hr />
+          <nav class="lists d-flex space-between">
```

Comment:
- [ ] Kindly change the `nav` tag to the `div` because there is no sense using the `nav` tag. And also you can use `nav` inside the `header` tag in order to make you code Semantic 🙏. In there you can easily use `div`.